staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler_test.go

```diff
@@ -755,3 +759,87 @@ func Test_defaultDeprecationWarning(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildOpenAPIModelsForApply(t *testing.T) {
+	// This is a list of validation that we expect to work.
+	tests := []apiextensionsv1.CustomResourceValidation{
```

Comment:
Can you test a nullable? Can we find a list of the differences between v3 and v2 and proactively see what could cause an error in the future?